src/pages/storage/forms/StorageVolumeFormMain.tsx

```diff
@@ -45,9 +32,26 @@ const StorageVolumeFormMain: FC<Props> = ({ formik, project }) => {
             help={
               formik.values.isCreating
                 ? undefined
-                : "Click the name in the header to rename the volume"
+                : "Click the name in the header to rename the volume."
             }
           />
+
+          <Label forId="storage-pool-selector" required>
+            Storage pool
+          </Label>
+          <StoragePoolSelector
+            project={project}
+            value={formik.values.pool}
+            setValue={(val) => void formik.setFieldValue("pool", val)}
+            hidePoolsWithUnsupportedDrivers
+            disabled={!formik.values.isCreating}
```

Comment:
There is no required flag set in the storagePoolSelector component and therefore I am assuming it is already false. Please could you clarify what you would like me to change?

Reply:
Sorry, that was not clear enough, the required is on the label above, not inside the selector component.

Reply:
Right, but you said you want the selector component to have required set to false, I am asking, isn't that the default behaviour? As it has not been set at all in that component.

Reply:
I think this is still confused. What I meant is:

The label of the pool selector has a * in it, because it is a required label. It should keep the * during creation. Now that we also show it on edit as disabled, it should be the label without that *, meaning removing the required flag from the label in the edit case. The selector itself is fine and doesn't have a required flag, as it doesn't come with the label included.